mmv1/third_party/terraform/services/container/resource_container_cluster_test.go.tmpl

```diff
@@ -6297,6 +6297,83 @@ func testAccCheckContainerClusterDestroyProducer(t *testing.T) func(s *terraform
 	}
 }
 
+{{ if ne $.TargetVersionName `ga` -}}
+func TestAccContainerCluster_withPodAutoscalingConfig(t *testing.T) {
+	t.Parallel()
+
+	networkName := "gke-cluster-pod-hpa"
+	subnetworkName := "gke-cluster-pod-hpa"
+	clusterName := fmt.Sprintf("tf-test-cluster-%s", acctest.RandString(t, 10))
+	pid := envvar.GetTestProjectFromEnv()
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:     func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderBetaFactories(t),
+		CheckDestroy: testAccCheckContainerClusterDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccContainerCluster_withPodAutoscalingConfig(pid, networkName, subnetworkName, clusterName, true),
+			},
+			{
+				ResourceName:      "google_container_cluster.with_pod_autoscaling_config",
+				ImportState:       true,
+				ImportStateVerify: true,
+				ImportStateVerifyIgnore: []string{"deletion_protection"},
+				Check: resource.TestCheckResourceAttr(
+					"google_container_cluster.with_pod_autoscaling_config", "pod_autoscaling.hpa_profile", "HPA_PROFILE_UNSPECIFIED"),
+			},
+			{
+				Config: testAccContainerCluster_withPodAutoscalingConfig(pid, networkName, subnetworkName, clusterName, false),
```

Comment:
Update tests should include ConfigPlanChecks on all the Config steps except the first one, as described at https://googlecloudplatform.github.io/magic-modules/test/test/#add-an-update-test